quilt/tools/command.py

```diff
@@ -489,18 +489,27 @@ def install(package, hash=None, version=None, tag=None, force=False):
     pkgobj = store.install_package(owner, pkg, response_contents)
 
     total = len(response_urls)
-    for idx, (download_hash, url) in enumerate(iteritems(response_urls)):
-        print("Downloading object %d/%d..." % (idx + 1, total))
+    for idx, (download_hash, url) in enumerate(sorted(iteritems(response_urls))):
+        print("Downloading %s (%d/%d)..." % (download_hash, idx + 1, total))
+
+        local_filename = store.object_path(download_hash)
+        if os.path.exists(local_filename):
+            file_hash = digest_file(local_filename)
+            if file_hash == download_hash:
+                print("Object already exists; skipping.")
```

Comment:
"Object" is a loaded word. Can we use "fragment" or "file"?

We should also use quilt vocabulary wherever possible:
"Fragment already installed; skipping"